R/sjTabItemAnalysis.R

```diff
@@ -139,6 +139,12 @@ sjt.itemanalysis <- function(df,
   # check encoding
   encoding <- get.encoding(encoding, df)
 
+  # convert ordered factors to labbeled
+  if (all(sapply(df, is.ordered))) df <- sjlabelled::as_labelled(df)
+
+  # Warn if factors are used
+  if (all(sapply(df, is.factor))) stop("Factors are not allowed. Please check if factor levels are ordered and use sjlabelled::as_labbeled() for conversion.")
```

Comment:
1) If factors are not allowed, either check with `any`, and probably also convert to numeric automatically?
2) typo in sjlabelled::as_labbeled()